components/service/pocket/src/test/java/mozilla/components/service/pocket/PocketStoriesServiceTest.kt

```diff
@@ -34,14 +40,76 @@ class PocketStoriesServiceTest {
     fun `GIVEN PocketStoriesService WHEN startPeriodicStoriesRefresh THEN scheduler#schedulePeriodicRefreshes should be called`() {
         service.startPeriodicStoriesRefresh()
 
-        verify(service.scheduler).schedulePeriodicRefreshes(any())
+        verify(service.storiesRefreshScheduler).schedulePeriodicRefreshes(any())
     }
 
     @Test
     fun `GIVEN PocketStoriesService WHEN stopPeriodicStoriesRefresh THEN scheduler#stopPeriodicRefreshes should be called`() {
         service.stopPeriodicStoriesRefresh()
 
-        verify(service.scheduler).stopPeriodicRefreshes(any())
+        verify(service.storiesRefreshScheduler).stopPeriodicRefreshes(any())
+    }
+
+    @Test
+    fun `GIVEN PocketStoriesService initialized with a spocs profile WHEN startPeriodicSponsoredStoriesRefresh THEN schedule periodic refreshes`() {
```

Comment:
nit: is initialized

nit: It's better not to encode method names directly into test names otherwise. WHEN perdioc refresh starts THEN...
(This applies to a few tests here.)

Reply:
Agree. Sometimes though it's more easier to reason about the exact "unit" / method tested or just takes fewer words to express that but I'll keep this is mind, thank you!